test/testUtils/TransferMockHelper.scala

```diff
@@ -0,0 +1,74 @@
+package testUtils
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock.{equalToJson, okJson, post, urlEqualTo}
+import configuration.GraphQLConfiguration
+import graphql.codegen.AddFinalTransferConfirmation.{addFinalTransferConfirmation => aftc}
+import graphql.codegen.UpdateTransferInitiated.{updateTransferInitiated => ut}
+import io.circe.Printer
+import io.circe.generic.auto._
+import io.circe.syntax._
+import play.api.Application
+import uk.gov.nationalarchives.tdr.GraphQLClient
+
+import java.util.UUID
+import scala.concurrent.ExecutionContext
+
+object TransferMockHelper {
```

Comment:
We already have `FrontEndTestHelper` so you can move these stubs to this class.

Reply:
moved